joomla/joomla-cms
# [[5.x]Compatibility Plugin (BC) added to system info

### Summary of Changes
It's going to be more and more important to be able to know if the Backward Compatibility Plugin is enabled or not.

This PR adds it to the list of information displayed in the system information page


### Testing Instructions
Apply the PR and you will now be able to see if the plugin is enabled or disabled when you so the system dashboard and click on system information

![image](https://github.com/joomla/joomla-cms/assets/1296369/84fc5849-8692-4add-be0f-dfb9003bca2f)




### Link to documentations
Please select:
- [] Documentation link for docs.joomla.org: <link>
